src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  let sumInDays = days * 40;
+
+  if (days >= 7) {
+    sumInDays -= 50;
+  } else if (days >= 3) {
+    sumInDays -= 20;
```

Comment:
Let's follow a checklist to improve your code a little bit)

<img width="1043" alt="Screenshot 2023-04-01 at 19 00 10" src="https://user-images.githubusercontent.com/32853034/229304455-33fdf356-2362-4d9c-904f-151e1db48cc9.png">